internal/output/markdown.go

```diff
@@ -140,6 +186,14 @@ func ToMarkdown(out Root, opts Options, markdownOpts MarkdownOptions) ([]byte, e
 			}
 			return true
 		},
+		"formatEmissions": func(d *decimal.Decimal) string { return formatEmissions(d, "kgCO2e") },
+		"formatEmissionsChange": func(pastCost, cost *decimal.Decimal) string {
+			return formatMarkdownEmissionsChange(pastCost, cost)
+		},
+		"formatEmissionsChangeSentence": formatEmissionsChangeSentence,
+		"formatFields": func(fields []string) string {
+			return strings.Join(fields, ", ")
+		},
```

Comment:
I didn't find where this is used.

Reply:
```suggestion
```
Indeed it is no longer used in the final version !